webapp/packages/core-connections/src/DatabaseAuthDialog/DBAuthDialogFooter.tsx

```diff
@@ -9,41 +9,68 @@
 import { observer } from 'mobx-react-lite';
 import styled, { css } from 'reshadow';
 
-import { Button } from '@cloudbeaver/core-blocks';
+import { Button, ErrorMessage } from '@cloudbeaver/core-blocks';
 import { useTranslate } from '@cloudbeaver/core-localization';
+import type { GQLErrorCatcher } from '@cloudbeaver/core-sdk';
+import { composes, useStyles } from '@cloudbeaver/core-theming';
 
-const styles = css`
-  controls {
-    display: flex;
-    height: 100%;
-    flex: 1;
-    align-items: center;
-    margin: auto;
-    justify-content: flex-end;
-  }
-`;
+const styles = composes(
+  css`
+    ErrorMessage {
+      composes: theme-background-secondary from global;
+    }
+  `,
+  css`
+    controls {
+      display: flex;
+      height: 100%;
+      flex: 1;
+      align-items: center;
+      justify-content: flex-end;
+    }
+    ErrorMessage {
+      flex: 0.9;
+      & + controls {
+        margin-left: 17px;
+        flex: 0.1;
+      }
```

Comment:
strange flex values

Reply:
also, another problem is, that ErrorMessage and controls have no parent with `display: flex` so we can't guarantee that component that will use `DBAuthDialogFooter` wrap this component with element with `display: flex`